litecli/packages/special/dbcommands.py

```diff
@@ -86,10 +86,9 @@ def show_schema(cur, arg=None, **_):
     if cur.description:
         headers = [x[0] for x in cur.description]
     else:
-        return [(None, None, None, "")]
-
-    return [(None, tables, headers, status)]
+        return [(None, None, None, "")] + _list_indexes(cur, arg=arg)
 
+    return [(None, tables, headers, status)] + _list_indexes(cur, arg=arg)
```

Comment:
As per my understanding, `l1 + l2`or `l1.extend(l2)`or [*l1, l2]` have similar performance hence used the most intutive one.